README.md

```diff
@@ -1,339 +1,108 @@
 # nextcloud-dev-docker-compose
 
-Nextcloud development environment using docker-compose
+Hi beginner developer ! 👋
 
-⚠ **DO NOT USE THIS IN PRODUCTION** Various settings in this setup are considered insecure and default passwords and secrets are used all over the place
```

Comment:
I'd love to keep this note as people should never expose this environment to the public or run it anywhere except their local and hopefully secured machines.

Reply:
Okay, I rewrite your text in the README.md :)

I specify that this project is for the development environment only here : https://github.com/arawa/nextcloud-docker-dev/blob/66251329279419ebaf5d69147240c1e55dec24d5/README.md?plain=1#L8

Maybe is insufficient ? :thinking:

Reply:
I fixed your request in this commit : https://github.com/juliushaertl/nextcloud-docker-dev/pull/87/commits/7442538c2f81bd270dbb58acec2c691b3c654918